tests/TestCase.php

```diff
@@ -13,20 +13,21 @@
 use App\Http\Controllers\Backend\Transport\TrainCheckinController;
 use App\Http\Controllers\TransportController;
 use App\Models\Event;
-use App\Models\HafasTrip;
-use App\Models\TrainStation;
 use App\Models\User;
+use App\Http\Controllers\Backend\WebhookController;
+use App\Models\Webhook;
+use App\Repositories\OAuthClientRepository;
 use Carbon\Carbon;
 use Exception;
 use Illuminate\Database\Eloquent\ModelNotFoundException;
 use Illuminate\Foundation\Testing\TestCase as BaseTestCase;
 use Illuminate\Support\Collection;
 use Illuminate\Testing\TestResponse;
 use JetBrains\PhpStorm\ArrayShape;
+use Laravel\Passport\Client as PassportClient;
 use Tests\Feature\CheckinTest;
 
-abstract class TestCase extends BaseTestCase
-{
+abstract class TestCase extends BaseTestCase {
```

Comment:
Can you create a test which tests the creation of an oauth client and token, that also checks if the webhook is created successfully? That would allow us to have a documented way how the flow should work.

Reply:
The OAuth flow is really complicated to test and I think it would be way easier (and easier to maintain) to do this in an actual browser e2e test rather than in an integration test.
I have not really found a way to do this in an integration test without touching passport internals.

Reply:
Okay no worries, we can add that later as well.